packages/create-cloudflare/src/common.ts

```diff
@@ -562,7 +560,7 @@ function prepareCommitMessage(commitMessage: string): string {
 
 export function quoteShellArgs(args: string[]): string {
 	if (process.platform !== "win32") {
-		return quote(args);
+		return args.join(" ");
```

Comment:
I feel bad that we are now doing a more simplistic quoting for non-Windows that for Windows.

Could we consider implementing this simple snippet for each `s` of `args` from the shell-quote library, which I think covers most of our concerns?

```js
		if ((/["\s]/).test(s) && !(/'/).test(s)) {
			return "'" + s.replace(/(['\\])/g, '\\$1') + "'";
		}
		if ((/["'\s]/).test(s)) {
			return '"' + s.replace(/(["\\$`!])/g, '\\$1') + '"';
		}
```